package.json

```
@@ -45,6 +45,7 @@
     "test:lint:css": "stylelint 'src/**/*.js' 'src/**/*.jsx'",
     "test:unit": "jest --env=jsdom --coverage --colors",
     "snyk": "snyk test",
+    "spartacus:integrationTest": "npm run build && npm run test && xvfb-run npm run test:e2e:storybook:ci && xvfb-run npm run test:prod:ci",
```

```suggestion
    "spartacus:integrationTest": "npm run build && npm run test && xvfb-run npm run test:e2e:storybook:ci && xvfb-run npm run test:prod:ci && xvfb-run npm run test:lighthouse:ci",
```

We should include the lighthouse tests in the Spartacus travis integration tests. Currently, these fail due to the way we are using `npm link` to test this. We may have to look into alpha publishing to properly mock the integration of Spartacus with upstream applications.